src/hct_mis_api/config/settings.py

```diff
@@ -500,7 +503,7 @@ def masker(key: str, value: Any, config: Dict, request: HttpRequest) -> Any:
 from hct_mis_api.config.fragments.social_auth import *  # noqa: F403, F401, E402
 from hct_mis_api.config.fragments.storages import *  # noqa: F403, F401, E402
 
-LIBRARY_PATHS: bool = env("LIBRARY_PATHS", default=False)
```

Comment:
I'm not sure if removing this particular default will benefit us. It's used only for local selenium testing.

Now on every deployed instance, we'll have to explicitly set this to false.

Reply:
Oh, nevermind. The default is set in the `env.py`.
```
"LIBRARY_PATHS": (bool, False),
```

Well then this still will work.
But I wouldn't say moving the default there makes it more readable when viewing the code :P